user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -19,111 +19,295 @@ redirect_from:
+    <i class="fa fa-question-circle"></i>
+    <b>Note : </b> Like Fedora, Qubes OS uses the Anaconda installer. Those that are familiar with RPM-based distributions should feel at home. 
+</div>
+
+<h3>Installation summary</h3>
+
+The Installation summary screen allows you to change how the end-system will be installed and configured, including localization settings. 
+At minimum, you are required to pick a storage device on which Qubes OS will be installed. 
+
+<img src="/attachment/wiki/InstallationGuide/installation-summary-not-ready.png">
+
+<h3>Localization</h3>
+
+Let's assume you wish to add a German keyboard layout. Go to Keyboard Layout, press the "Plus" symbol, search for "German" as indicated in the screenshot 
+and press "Add". If you want it be your default language, select the "German" entry in the list and press the arrow button. Click on "Done" in the upper left corner and you are ready to go ! 
+
+<img src="/attachment/wiki/InstallationGuide/keyboard-layout-selection.png">
+
+The process to select a new language is similar to the process to select a new keyboard layout. Follow the same process in the "Language Support" entry.
+
+<img src="/attachment/wiki/InstallationGuide/language-support-selection.png">
+
+<div class="alert alert-info" role="alert">
+    <i class="fa fa-question-circle"></i>
+    <b>Note : </b> You can have as many keyboard layout and languages as you want. Post-install, you will be able to switch between them and install others. 
+</div>
+
+Don't forget to select your time and date by clicking on the Time & Date entry.
+
+<img src="/attachment/wiki/InstallationGuide/time-and-date.png">
+
+<h3>Software</h3>
+
+Under the Software section, you can change the installation source. As we are demonstrating a simple installation, it is assumed that you are installing Qubes OS using 
+a local medium such as a DVD, so this option won't be illustrated.
 
-Before proceeding with the installation, you are encouraged to first read all
-the information on this page. When you're ready, boot your system from the
-installation source and follow the on-screen instructions. The installer is very
-simple and asks very few questions. (It's actually easier to install Qubes right
-now than most other Linux distributions!)
+<img src="/attachment/wiki/InstallationGuide/add-ons.png">
 
-The installer loads Xen right at the beginning, so chances are high that if you
-can see the installer's graphical screen and you pass the compatibility check that
-runs immediately after that, Qubes will work on your system. :)
+Go instead to the Software selection tab, where you can choose which software to install alongside Qubes OS. Two Add-Ons are available :
 
-If you're an advanced user, and you'd like to customize your installation, please see [Custom Installation].
+* **Debian 9 (stretch) template** : Install these templates if you wish to base some of your Qubes virtual machines on Debian instead of Fedora. 
```

Comment:
It might be better to skip the version in here, as it varies between different releases. For example Qubes R4.0.2 will ship with Debian 10 (buster).

Reply:
Just using Debian instead of Debian 9